treeTable.js

```diff
@@ -118,21 +116,46 @@
 
     const TreeTable = function (element, options) {
         const self = this;
+        this.$el = $(element);
+        if (options.data) {
+            this.init(options);
+        } else if (options.ajax) {
+            this.$dummy = $("<table></table>");
```

Comment:
So this uses DataTable's ajax support, to load ajax data into a dummy DT then handle the event on completion of that load to clean up the dummy table and set its own data with data built from the loaded raw json plus settings, using DT's internal method `_fnAjaxDataSrc`? 
If not, then this could probably use a comment! If so, I still wouldn't object to a brief comment on general strategy and what `_fnAjaxDataSrc` is!

Reply:
Yes, you got it, but will add comment!